core/unit_test/TestExecSpaceThreadSafety.hpp

```diff
@@ -78,11 +78,9 @@ void run_exec_space_thread_safety_range() {
 
 TEST(TEST_CATEGORY, exec_space_thread_safety_range) {
 #ifdef KOKKOS_ENABLE_OPENACC  // FIXME_OPENACC
-#ifdef KOKKOS_ENABLE_OPENACC_FORCE_HOST_AS_DEVICE
   if (std::is_same_v<TEST_EXECSPACE, Kokkos::Experimental::OpenACC>)
     GTEST_SKIP() << "skipping since test is known to fail with OpenACC";
```

Comment:
Do you want to say something like race condition in shared allocation reference counting or host-side atomics?

Reply:
Added more detailed error message.